public/assets/js/splash.js

```diff
@@ -0,0 +1,35 @@
+let alreadyShowed = showedCookieExists();
+let splash = document.querySelector("#splash");
+let studioName = document.querySelector(".studio-name");
+let nameFragment = document.querySelectorAll(".name-fragment");
+
+function showed() {
+    var data = new Date();
+    data.setMinutes(data.getMinutes() + 10);
+    data = data.toGMTString();
+    document.cookie = "showed=true" + "; expires=" + data + "; path=/";
+}
+
+function showedCookieExists() {
+    var cname = "showed" + "=";
+    var cookies = document.cookie;
+
+    return cookies.indexOf(cname) !== -1;
+}
+
+window.addEventListener("DOMContentLoaded", () => {
+    if (!alreadyShowed) {
```

Comment:
Acho que esse ponto se você fizesse dessa forma:
```suggestion
    if (!alreadyShowed) {
         console.log("Teste")
    }
```